lib/LedgerSMB/Scripts/asset.pm

```diff
@@ -1010,7 +1010,18 @@ sub run_import {
 }
 
 ###TODO-LOCALIZE-DOLLAR-AT
-eval { do "scripts/custom/asset.pl"};
+eval {
```

Comment:
Ok, I'll strip them out. I suspected that was the case, but wasn't certain so left them behind.